docs/device-management-test.md

```diff
@@ -57,7 +57,7 @@ Mbed Device Management tests rely on the Python SDK to test the end-to-end solut
 
 ### Testing setup
 
-1. Import an example application for Pelion Device Management that contains the corresponding configuration for your target. 
+1. Import an example application for Device Management that contains the corresponding configuration for your target.
 
    Please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
```

Comment:
Is this really meant to be a reference to the entire platform? That include services outside Device Management.

Reply:
Do you mean the term "Pelion IoT Platform service" includes services outside Device Management, or the example does? If the former, probably not. What should it be instead?

Reply:
Pelion IoT Platform includes services outside of Device Management. Does the examples include/use those outside services?

Reply:
If not, it should just be Device Management.